### PR TITLE
quickstart: remove deprecated disk handling

### PR DESCRIPTION
**Pull request by Lassulus**
this should take care of https://github.com/nix-community/disko/issues/429

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
-### Step 5: Run disko to partition, format and mount your disks
+### Step 5: Adjust the device in the disk configuration
 
-The following step will partition and format your disk, and mount it to `/mnt`.
+Inside the disko-config.nix the device needs to point to the correct disk name.
 Replace `<disk-name>` with the name of your disk obtained in Step 1.
 
-Please note: This will erase any existing data on your disk.
+```
+nano disko-config.nix
```
Comment (phaer) [inline docs/quickstart.md:87]:
```suggestion
nano /tmp/disko-config.nix
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
+      main = {
+        type = "disk";
+        device = "<disk-name>";
+        content = {
+          type = "gpt";
+...
 ```
 
-For example, if the disk name is `nvme0n1`, the command would be:
+### Step 5: Run disko to partition, format and mount your disks
```
Comment (phaer) [inline docs/quickstart.md:100]:
```suggestion
### Step 6: Run disko to partition, format and mount your disks
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
+
+The following step will partition and format your disk, and mount it to `/mnt`.
+
+Please note: This will erase any existing data on your disk.
```
Comment (phaer) [inline docs/quickstart.md:104]:
```suggestion
**Please note: This will erase any existing data on your disk.**
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -78,23 +78,36 @@ cd /tmp
 $ curl https://raw.githubusercontent.com/nix-community/disko/master/example/hybrid.nix -o disko-config.nix
```
Comment (phaer) [inline docs/quickstart.md:78]: maybe remove the cd /tmp above, to make it a bit more stateless? :)

```suggestion
$ curl https://raw.githubusercontent.com/nix-community/disko/master/example/hybrid.nix -o /tmp/disko-config.nix
```

File: docs/quickstart.md (hunk 2 above)
Comment (phaer) [reply docs/quickstart.md:100]: Need to adapt all the other step numbers as well, or just remove the numbering? :)

**Review by phaer — APPROVED** [review]

File: docs/quickstart.md [hunk 5]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
 
-### Step 5: Run disko to partition, format and mount your disks
+### Step 5: Adjust the device in the disk configuration
 
-The following step will partition and format your disk, and mount it to `/mnt`.
+Inside the disko-config.nix the device needs to point to the correct disk name.
 Replace `<disk-name>` with the name of your disk obtained in Step 1.
 
-Please note: This will erase any existing data on your disk.
+```
```
Comment (Mic92) [inline docs/quickstart.md:86]:
```suggestion
Open the configuration in your favorite editor i.e.:

```
```

File: docs/quickstart.md [hunk 6]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
+nano disko-config.nix
+```
 
 ```
```
Comment (Mic92) [inline docs/quickstart.md:90]:
```suggestion
```

Adjust `<disk-name>` with the name of your disk:

```

File: docs/quickstart.md [hunk 7]
```diff
@@ -78,23 +78,36 @@ cd /tmp
...
+### Step 5: Adjust the device in the disk configuration
 
-The following step will partition and format your disk, and mount it to `/mnt`.
+Inside the disko-config.nix the device needs to point to the correct disk name.
 Replace `<disk-name>` with the name of your disk obtained in Step 1.
 
-Please note: This will erase any existing data on your disk.
+```
+nano disko-config.nix
+```
```
Comment (Mic92) [inline docs/quickstart.md:89]:
```suggestion

Adjust `<disk-name>` with the name of your disk:

```

File: docs/quickstart.md [hunk 8]
```diff
@@ -78,23 +78,36 @@ cd /tmp
 $ curl https://raw.githubusercontent.com/nix-community/disko/master/example/hybrid.nix -o disko-config.nix
 ```
 
-### Step 5: Run disko to partition, format and mount your disks
+### Step 5: Adjust the device in the disk configuration
 
-The following step will partition and format your disk, and mount it to `/mnt`.
+Inside the disko-config.nix the device needs to point to the correct disk name.
 Replace `<disk-name>` with the name of your disk obtained in Step 1.
```
Comment (Mic92) [inline docs/quickstart.md:84]:
```suggestion

```

File: docs/quickstart.md (hunk 7 above)
Comment (Mic92) [inline docs/quickstart.md:89]:
```suggestion

Replace `<disk-name>` with the name of your disk obtained in Step 1.

```

**Comment by Lassulus** [comment]
@mergify queue

**Pull request merged**